SU-SWS/stanford_person
# HSDO-840 Recovered missing variable by rebuilding the feature correctly

# READY FOR REVIEW

# Summary
- Recover variable that allows the configuration of regions on content types.

# Needed By (Date)
- 5/10

# Steps to Test

1. Checkout branch revert feature
2. check page admin/structure/types/manage/stanford_person/display/full to validate the regions are there: "Office Hours", "Contact", "Research Interests"
